testing/mldb_python_plugin_test.cc

```diff
@@ -44,13 +44,13 @@ BOOST_AUTO_TEST_CASE( test_python_loading )
     scriptConfig["address"] = "file://mldb/testing/python_script_test1.py";
 
     auto output = proxy.post("/v1/types/plugins/python/routes/run", scriptConfig);
-    BOOST_CHECK_EQUAL(output.code(), 200);
+    BOOST_REQUIRE_EQUAL(output.code(), 200);
```

Comment:
I've updated BOOST_CHECK to BOOST_REQUIRE, which makes it easier to debug errors. The downside is that it makes a lot of noise here. I'll highlight the lines I did "really" change for that PR.